tests/DependencyInjection/Fixtures/config/xml/full.xml

```diff
@@ -86,20 +86,96 @@
                     tlsDisableOCSPEndpointCheck="false"
                 />
                 <doctrine:encryptedFieldsMap>
-                    <doctrine:encryptedFields name="encrypted.patients">
-                        <doctrine:field path="patientRecord.ssn" bsonType="string">
-                            <doctrine:queries queryType="equality" />
-                        </doctrine:field>
-                        <doctrine:field path="patientRecord.billing" bsonType="object" />
-                        <doctrine:field path="patientRecord.billingAmount" bsonType="int">
-                            <doctrine:queries queryType="range" min="100" max="2000" sparsity="1" trimFactor="4" />
-                        </doctrine:field>
-                    </doctrine:encryptedFields>
-                    <doctrine:encryptedFields name="encrypted.users">
-                        <doctrine:field path="email" bsonType="string">
-                            <doctrine:queries queryType="equality" />
-                        </doctrine:field>
-                    </doctrine:encryptedFields>
+                    <![CDATA[
```

Comment:
@alcaeus Since the XML format is set to disappear, I have simplified the configuration here. EncryptedFieldsMap is a JSON string that is copied from the dump command in the XML. It is too complicated to convert this to XML tags, and most likely unusable anyway.

Reply:
That makes sense 👍